.travis.yml

```diff
@@ -1,106 +1,22 @@
 sudo: false
 
-language: generic
-
-dist: precise
+language: node_js
+node_js:
+ - "node"
+ - "lts/*"
+ - "7"
+ - "6"
+ - "5"
+ - "4"
```

Comment:
I've left out 0.10 and 0.12 on purpose since Node.js is in a much quicker release cycle and the latest LTS major version is already 8.